### ISSUE TITLE
Fig paths suggestions stop working | ZSH | Hyper | cd 

### ISSUE THREAD

**Issue by ShawnCockburn** [withfig/fig#635]
### Description:
> Please include a detailed description of the issue (and an image or screen recording, if applicable)

fig works for command suggestions but fails to show path (files folders) in auto complete.
I tried the fix for the last fig issue I had: https://github.com/withfig/fig/issues/476#issuecomment-934680931
but that does not fix the issue, only full restart works.
not sure how to reproduce as the issue happens randomly. 
[YouTube video showing the issue](https://www.youtube.com/watch?v=DseKIZwEGVM)

### Details:
|macOS|Fig|Shell|
|-|-|-|
|12.0.1.21A559|1.0.52|/bin/zsh|
<details><summary><code>fig diagnostic</code></summary>
<p>
<pre>Fig Version: Version 1.0.52 (B317) [British] 
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /usr/local/bin/fig
Accessibility: true
Number of specs: 204
SSH Integration: false
Tmux Integration: false
Keybindings path: /Users/Work/.fig/user/keybindings
iTerm Integration: application is not present.
Hyper Integration: installed!
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/Work/.fig/bin
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: /bin/zsh (49330) - ttys008
Current working directory: /Users/Work
Current window identifier: 107252/% (co.zeit.hyper)
Path: /usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin:/Users/Work/.fig/bin</pre>
</p>
</details>

**Comment by ShawnCockburn** [comment]
### Update: 

so It seems that it was not as random as I thought, it gets triggered by using fig on a separate user like my last issue: 

these commands remedy the issue without restarting the system
```bash
killall fig

rm /tmp/fig.socket

killall 'zsh (figterm)'

fig
```

any ideas on the cause?

**Comment by brendanfalk** [comment]
@ShawnCockburn is this issue still happening? Try updating to the newest version (◧ > Check for Updates). I am also going to merge this issues I am about to create

**Comment by ShawnCockburn** [comment]
@brendanfalk, I just updated and its still happening. 

the steps to reproduce.

cd to a dir eg documents/ `cd Documents/` and then cd .. `cd ..` NOT  cd../' . this always reproduces the issue for me

**Comment by NoirJ0e** [comment]
I can confirm that this problem also happened in zsh with iTerm2

the output of `fig diagnostic`:
Fig Version: Version 1.0.53 (B339) [Squirrel]
UserShell: /bin/zsh
Bundle path: /Applications/Fig.app
Autocomplete: true
Settings.json: true
CLI installed: true
CLI tool path: /Users/noirjoe/.fig/bin/fig
Accessibility: true
Number of specs: 0
SSH Integration: false
Tmux Integration: false
Keybindings path: <none>
iTerm Integration: installed!
Hyper Integration: application is not present.
VSCode Integration: installed!
Docker Integration: false
Symlinked dotfiles: false
Only insert on tab: false
Installation Script: true
PseudoTerminal Path:
SecureKeyboardInput: false
SecureKeyboardProcess: <none>
Current active process: zsh (13255) - /dev/ttys004
Current working directory: /Users/noirjoe
Current window identifier: 2984/% (com.googlecode.iterm2)
Path: /usr/local/opt/qt@5/bin:/usr/local/opt/curl/bin:/usr/local/opt/coreutils/libexec/gnubin:/usr/local/Cellar/zplug/2.4.2/bin:/usr/local/opt/zplug/bin:/Users/noirjoe/.fig/bin:/usr/local/opt/qt@5/bin:/usr/local/opt/curl/bin:/usr/local/opt/coreutils/libexec/gnubin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/opt/zplug/repos/unixorn/fzf-zsh-plugin/bin:/Users/noirjoe/.fzf/bin:/usr/local/opt/zplug/repos/unixorn/fzf-zsh-plugin/bin
Fig environment variables:
  - TERM_SESSION_ID=w0t0p0:9FEC17DE-2715-4E21-A39E-3778DBEA49F3
  - FIG_INTEGRATION_VERSION=5
  - FIG_CHECKED_PROMPTS=1
  - FIG_TERM=1
  - FIG_TERM_VERSION=3
  - FIG_ENV_VAR=1

**Comment by ShawnCockburn** [comment]
i'm closing this, seems to be fixed.

@NoirJ0e are you still experiencing this?

**Comment by NoirJ0e** [comment]
> i'm closing this, seems to be fixed.
> 
> @NoirJ0e are you still experiencing this?

Unfortunately yes, I don't have any idea about this, whatever the I use the cd command or  [zoxide](https://github.com/ajeetdsouza/zoxide)

**Comment by ShawnCockburn** [comment]
> > i'm closing this, seems to be fixed.
> > @NoirJ0e are you still experiencing this?
> 
> Unfortunately yes, I don't have any idea about this, whatever the I use the cd command or [zoxide](https://github.com/ajeetdsouza/zoxide)

It seems to be fixed for me after the last update (1.0.54 B359), I'm sorry to hear your still having an issue mate :( 

![This helped me](https://github.com/withfig/fig/issues/476#issuecomment-934680931) with a similar issue, however not this one; maybe it'll help you.

if you haven't already then open a new issue for the devs, @mschrage is very helpful.

`fig issue`

good luck!

**Comment by mschrage** [comment]
@NoirJ0e can you create a new issue? I'd love to get Fig working for you :)
